django/create_configuration.py

```diff
@@ -49,7 +49,7 @@ def backup_existing_copy(path):
 out_configfile = op.join('projects/mysite/django.wsgi')
 backup_existing_copy(out_configfile)
 
-o = open( out_configfile ,'w')
+o = open( out_configfile,'w')
```

Comment:
I think the removed space belongs after the comma in front of the `'w'`.

Reply:
And the space after `open(` can be removed, too.

Reply:
Whitespaces in and around arguments are some of the rules I removed because there are so many infractions, but you're right that lines we're changing anyway should be fixed so that we can maybe incorporate it in future.

Reply:
I'll also replace these explicit open/close bits with `with` blocks, in line with best practice.

Reply:
:+1: